tests/integration/suite/daprd/pubsub/grpc/appready.go

```diff
@@ -143,12 +143,14 @@ func (a *appready) Run(t *testing.T, ctx context.Context) {
 		return resp.StatusCode == http.StatusOK
 	}, time.Second*5, 10*time.Millisecond)
 
-	_, err = client.PublishEvent(ctx, &rtv1.PublishEventRequest{
-		PubsubName: "mypubsub",
-		Topic:      "mytopic",
-		Data:       []byte(`{"status": "completed"}`),
-	})
-	require.NoError(t, err)
```

Comment:
This should never fail- what error are we seeing?

Reply:
I forgot to grab the build failure link, but was seeing an err here

Reply:
https://github.com/dapr/dapr/actions/runs/10355704565/job/28664159218?pr=7973

Reply:
actually, maybe that wasn't the exact same err, just the same test I happened to see in an open PR currently

Reply:
The link you shared doesn't have anything to do with this block.